code-studio/src/js/components/StudioCellEditor.jsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * @overview React component to allow for easy editing and creation of
+ * Studio Cells
+ * @see @cdo/apps/studio/cell
+ */
+/* global React */
+var constants = require('@cdo/apps/studio/constants');
+
+var avatarList = ["dog", "cat", "penguin", "dinosaur", "octopus",
+    "witch", "bat", "bird", "dragon", "squirrel", "wizard", "alien",
+    "ghost", "monster", "robot", "unicorn", "zombie", "knight",
+    "ninja", "pirate", "caveboy", "cavegirl", "princess", "spacebot",
+    "soccergirl", "soccerboy", "tennisgirl", "tennisboy"];
```

Comment:
Is this the same list as studioAvatarList in Grid.jsx? Should we share them somehow - maybe in studio/constants?